### PR TITLE
CAMEL-17879: remove deprecated component camel-hystrix

### PR DESCRIPTION
**Pull request by orpiske**

### CODE REVIEW

**Comment by orpiske** [comment]
It might be better to wait until the whole set of tests run on my own CI before merging, so I think it would be better to wait until tomorrow for merge.

**Review by davsclaus — APPROVED** [review]

**Comment by orpiske** [comment]
The build completely cleanly and, apparently, without any problems on my own CI tests. We should be good to merge this one.

**Pull request merged**
